CoreFindView/src/au/gov/asd/tac/constellation/views/find2/plugins/advanced/AdvancedSearchPlugin.java

```diff
@@ -230,8 +232,10 @@ protected void edit(GraphWriteMethods graph, PluginInteraction interaction, Plug
                         // If the serach is select all and match critera = any
                         if (selectAll && ANY.equals(allOrAny)) {
 
+                            //LOGGER.log(Level.SEVERE, "INSIDE SELECT ANY");
```

Comment:
can you remove this line? I assume the logger imports were also for this so if that's the case, if you're able to remove those as well that would be great